docs/scheduling.txt

```diff
@@ -245,7 +245,11 @@ make upsmon set the "forced shutdown" (FSD) flag on the upsd so your
 slave systems shut down early too.  Just do something like this in your
 CMDSCRIPT:
 
-	/usr/local/ups/sbin/upsmon -c fsd
+	/sbin/upsmon -c fsd
+
+NOTE: the path to upsmon must be provided. The default for a source installation
+is /usr/local/ups (so /usr/local/ups/sbin/upsmon), while packaged installation
+will generally comply to FHS (so /sbin/upsmon).
```

Comment:
Also, given that the indented blocks will be fixed-width, pathnames in the text should probably be in `backticks` for consistency.